data/NFTE/data.json

```diff
@@ -1,16 +1,19 @@
 {
-    "name": "NFTEarth",
+    "name": "NFTEarthOFT",
     "symbol": "NFTE",
     "decimals": 18,
-    "description": "NFTEarth is a marketplace for NFTs.",
+    "description": "NFTEarth is a decentralized NFT marketplace on built on Optimism and other networks designed for scalability.",
     "website": "https://nftearth.exchange",
     "twitter": "@NFTEarth_L2",
```

Comment:
after this line you will want to add:

`"nobridge": true,`

to clarify it's not a standard bridgable token.